routes/channel.js

```diff
@@ -213,5 +227,18 @@ function authRequired(req, res, next) {
 	next()
 }
 
+function allowOnlyCreator(req, res, next) {
+	const {
```

Comment:
I find this over-usage of deconstruction unintuitive; just use req.channel.creator and req.session.uid please

Reply:
done